changelogs/fragments/391-sphinx-init-ansible-docs-fallback.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - sphinx-init - generated ``conf.py`` will be set to try resolving intersphinx references to Ansible's ``devel`` docs (https://github.com/ansible-community/antsibull/pull/391).
```

Comment:
```suggestion
  - The ``conf.py`` generated by ``antsibull-docs sphinx-init`` will be set to try resolving intersphinx references to Ansible's ``devel`` docs instead of a concrete Ansible version (https://github.com/ansible-community/antsibull/pull/391).
```